src/newSeed/stage3.ts

```diff
@@ -33,16 +33,13 @@ export class Stage3 extends BaseStage {
       this.validationError(message);
       this.stageState.verified = false;
     } else {
-      this.persistData();
```

Comment:
don't you need ` this.stageState.verified = true;` here?

Reply:
Not really, it is called by the detach function and set there

Reply:
Not really, it's funny but the detached function also calls the verifyInputs() and persistData() functions and also sets the stage state to true. Since the proceed function also triggers the detached state, the stage state is set after the call to validate input is done in the detached state.